REFERENCE.md

```diff
@@ -15,14 +15,15 @@
 * [`postgresql::lib::java`](#postgresqllibjava): This class installs the postgresql jdbc connector.
 * [`postgresql::lib::perl`](#postgresqllibperl): This class installs the perl libs for postgresql.
 * [`postgresql::lib::python`](#postgresqllibpython): This class installs the python libs for postgresql.
-* [`postgresql::server`](#postgresqlserver): This installs a PostgreSQL server
+* [`postgresql::server`](#postgresqlserver)
```

Comment:
Is this a missing `@summary`?

Reply:
Not sure........
The summary is in place in the manifest

Reply:
Gonna close this a rerun for the now labelled PR.
See if that fixes it